config/config.exs

```diff
@@ -17,6 +17,24 @@ config :mantra, Mantra.Endpoint,
   pubsub: [name: Mantra.PubSub,
            adapter: Phoenix.PubSub.PG2]
 
+config :ueberauth, Ueberauth,
+  providers: [
+    github: {Ueberauth.Strategy.Github, []}
+  ]
+
+config :ueberauth, Ueberauth.Strategy.Github.OAuth,
+  client_id: System.get_env("GITHUB_CLIENT_ID"),
```

Comment:
I used `dotenv` in ruby but now that I'm trying diff languages I use direnv (`brew install direnv`). will check your directory's .envrc for env settings. I can add a sample .envrc to this PR